app/assets/javascripts/application.js

```diff
@@ -3,6 +3,94 @@
 //= require turbolinks
 //= require_tree .
 
+// videocall
+
+let remoteContainer;
+let canvasContainer;
+
+let handleFail = function (err) {
+	console.log("Error : ", err);
+};
+
+function addVideoStream(streamId) {
+	let streamDiv = document.createElement("div");
+	streamDiv.id = streamId;
+	streamDiv.style.transform = "rotateY(180deg)";
+	remoteContainer.appendChild(streamDiv);
+}
+
+function removeVideoStream(evt) {
+	let stream = evt.stream;
+	stream.stop();
+	let remDiv = document.getElementById(stream.getId());
+	remDiv.parentNode.removeChild(remDiv);
+	console.log("Remote stream is removed " + stream.getId());
+}
+
+function addCanvas(streamId) {
+	let canvas = document.createElement("canvas");
+	canvas.id = 'canvas' + streamId;
+	canvasContainer.appendChild(canvas);
+	let ctx = canvas.getContext('2d');
+	let video = document.getElementById(`video${streamId}`);
+
+	video.addEventListener('loadedmetadata', function () {
+		canvas.width = video.videoWidth;
+		canvas.height = video.videoHeight;
+	});
+
+	video.addEventListener('play', function () {
+		var $this = this;
```

Comment:
This seems old syntax, can you use `let`?

Reply:
Done